test/development/basic/hmr.test.ts

```diff
@@ -35,25 +35,6 @@ describe.each([
   })
   afterAll(() => next.destroy())
 
-  it('should show hydration error correctly', async () => {
```

Comment:
This is unrelated to HMR. It was testing hydration errors in Pages router which is now covered by the added test suite.